uber/site_sections/registration.py

```diff
@@ -896,48 +896,6 @@ def undo_new_checkin(self, session, id):
         attendee.checked_in = attendee.group = None
         raise HTTPRedirect('new?message={}', 'Attendee un-checked-in')
 
-    def shifts(self, session, id, shift_id='', message=''):
```

Comment:
This got moved to shifts_admin, where it makes more sense. It can still be navigated to from Registration as before.